### PR TITLE
fix api-doc related bugs test=develop test=document_preview

### PR DESCRIPTION
**Pull request by lijianshe02**
fix the following nine api example docs: paddle.fluid.layers.open_files, paddle.fluid.layers.read_file, paddle.fluid.layers.mul, paddle.fluid.layers.reorder_lod_tensor_by_rank, paddle.fluid.WeightNormParamAttr, paddle.fluid.layers.is_empty, paddle.fluid.layers.Print, paddle.fluid.layers.has_inf, paddle.fluid.layers.has_nan

屏幕快照 2019-05-20 下午3 44 58
屏幕快照 2019-05-16 下午8 26 18
屏幕快照 2019-05-22 上午10 59 14
屏幕快照 2019-05-16 下午8 27 25
屏幕快照 2019-05-16 下午8 27 34
屏幕快照 2019-05-16 下午8 28 18
屏幕快照 2019-05-17 下午3 31 51
屏幕快照 2019-05-17 下午3 32 31

屏幕快照 2019-05-16 下午8 25 42

### CODE REVIEW

**Review by shanyi15 — DISMISSED** [review]

File: python/paddle/fluid/layers/control_flow.py [hunk 1]
```diff
@@ -2012,8 +2017,34 @@ def _assert_in_rnn_block_(self, method):
                 method))
 
 
-@autodoc()
+#@autodoc()
```
Comment (chengduoZH) [inline python/paddle/fluid/layers/control_flow.py]: Please remove line 2020.

Comment (lijianshe02) [reply python/paddle/fluid/layers/control_flow.py]: ok

**Review by shanyi15 — DISMISSED** [review]

File: python/paddle/fluid/layers/control_flow.py [hunk 2]
```diff
@@ -169,12 +169,17 @@ def Print(input,
 
 
     Examples:
-
         .. code-block:: python
+           
+           import paddle.fluid as fluid
+           from paddle.fluid.layers.control_flow import Print
```
Comment (chengduoZH) [inline python/paddle/fluid/layers/control_flow.py]: Please remove this line.

File: python/paddle/fluid/layers/control_flow.py [hunk 3]
```diff
@@ -169,12 +169,17 @@ def Print(input,
...
+           
+           input = fluid.layers.data(name="input", shape=[4, 32, 32], dtype="float32")
+           Print(input, message = "The content of input layer:")
```
Comment (chengduoZH) [inline python/paddle/fluid/layers/control_flow.py]: `Print ` -> `fluid.layers. Print `

Comment (lijianshe02) [reply python/paddle/fluid/layers/control_flow.py]: ok

File: python/paddle/fluid/layers/control_flow.py [hunk 4]
```diff
@@ -172,7 +172,7 @@ def Print(input,
         .. code-block:: python
            
            import paddle.fluid as fluid
-           from paddle.fluid.layers.control_flow import Print
+           from paddle.fluid.layers import Print
```
Comment (chengduoZH) [inline python/paddle/fluid/layers/control_flow.py]: Please remove line 175.

File: python/paddle/fluid/layers/control_flow.py (hunk 2 above)
Comment (lijianshe02) [reply python/paddle/fluid/layers/control_flow.py]: ok

**Review by luotao1 — APPROVED** [review]
LGTM

**Review by shanyi15 — APPROVED** [review]

**Pull request merged**